doc/developer/maintainer-release-build.rst

```diff
@@ -0,0 +1,85 @@
+Release Build Procedure for FRR maintainers
+=========================================================
+
+1) Rename branch (if needed)
+
+::
+
+	git clone git@github.com:FRRouting/frr.git
```

Comment:
Please use 3 space indents per the docs style guide, and I would change the `::` to `.. code-block:: shell` for better syntax highlighting. This is also the wrong syntax for lists, please use `1.`, `2.` etc or take advantage of automatically numbered lists and just use `#.` (http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks). So all in all something like this:

```
1. Rename branch (if needed):

   .. code-block:: shell

      git clone ...
```